### DISCUSSION TITLE
Welcome to tenderjit Discussions!

### DISCUSSION THREAD

**Discussion by tenderlove** [tenderlove/tenderjit#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 64kramsystem** [comment]
Hi there!

I'm Sav! :wave:

I'm the tech lead (and several other roles) on Invisible things™ (:joy:) at @ticketsolve. I have a very generous (and progressive) agreement, which grants me 40% of my work hours to any open source/studies/tech writing of my choice.

I was recently researching JITs in the emulation field, when I came across this project; I've realized it is a great match (for many reasons)! We'll see where this leads :smile:. I very much appreciate your attention to the community :star_struck:

Aside this, I'm planning to contribute to the QEMU Rust subprojects, an area currently evaluated by the QEMU team(s). I believe memory safety is a mission... where it can be realistically applied :grin:
